SR2021/2021-01-19-league-1-absent.md

```diff
@@ -0,0 +1,31 @@
+---
+to: Teams who didn't attend session 1
+subject: Student Robotics League 1
+---
+
+Hello there!
+
+We noticed at least one of your teams was absent from our first league session last weekend. We wanted to reach out to see if there was any more support we could provide you, or questions we could answer?
+
+## Assistance
```

Comment:
This is more pull than push, so perhaps
```suggestion
## Resources
```

Also maybe push this section down so that the focus is on the mentoring?